projects/index.html

```diff
@@ -255,8 +305,97 @@ <h3>History of Developmental Evolution</h3>
   											</div>
                         </div>
 										</div> -->
-                      <p><strong>Check back later for updated information on our research projects.</strong></p>
-                  </section>
+			<p><strong>Check back later for updated information on our research projects.</strong></p>
+		</section>
+
+		<section>
+
+			{% include section-header.html title="Recent Github Commits" %}
+
+			<table id="#dtBasicExample" style="border-collapse: collapse;">
+
+				<p id="paragraph"></p>
+
+			</table>
+
+
+		</section>
+
+	</div>
+</div>
+
+<script>
+
+	axios.get('https://api.github.com/orgs/diging/events')
+	.then((response) => {
+		data = response.data;
+		console.log(data);
+		
+		for(let i = 0; i < Object.keys(data).length; i++)
+		{
+			
+			if("PushEvent".localeCompare(Object.values(data)[i]["type"]) == 0)
+			{
+
+				let rowAttr = "myTr" + i.toString();
+
+				let row = document.createElement("TR");
+				row.setAttribute("id", rowAttr);
+				document.getElementById("#dtBasicExample").appendChild(row);
+
+				for(let j = 0; j < 4; j++)
```

Comment:
what is this loop for? You seem to be only creating 4 TD elements. You don't need a loop for that, just create 4 tds